framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -399,6 +399,15 @@ protected GenericAnnotatedTypeFactory(BaseTypeChecker checker, boolean useFlow)
         // all other initialization is finished.
     }
 
+    /**
+     * Returns the boolean value whether the analysis should be flow-sensitive or not.
+     *
+     * @return boolean value of useFlow field.
```

Comment:
```suggestion
     * @return whether flow-sensitive type refinement should be used or not
```

That it is backed by a field is an implementation detail.